src/csharp-stellar-base/PaymentOperation.cs

```diff
@@ -3,37 +3,41 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using Stellar.Generated;
 using static Stellar.Preconditions;
 
 namespace Stellar
 {
     public class PaymentOperation : Operation
     {
         public KeyPair Destination { get; private set; }
-        public Generated.Asset Asset { get; private set; }
+        public Asset Asset { get; private set; }
         public long Amount { get; private set; }
 
-        private PaymentOperation(KeyPair destination, Generated.Asset asset, long amount)
+        private PaymentOperation(KeyPair destination, Asset asset, long amount)
         {
-            this.Destination = CheckNotNull(destination, "destination cannot be null.");
-            this.Asset = CheckNotNull(asset, "asset cannot be null.");
-            this.Amount = amount;
+            Destination = CheckNotNull(destination, "destination cannot be null.");
```

Comment:
Starting a sentence with a capital is ok :)

Reply:
consistency with what was already there

Reply:
Adding onto that, destination refers to the variable by the same name and capitalization

Reply:
you are right, just leave it for now